lib/act.js

```diff
@@ -9,8 +9,9 @@ var call;
 var config;
 var widget;
 var cache;
+var debug = require('debug')('fh-mbaas-api:act');
```

Comment:
I wrote above before your comment @wojta.
if debug can output timestamps, I'd be in favour of the approach already in the PR